test/system/tests/createak.sh

```diff
@@ -54,6 +54,11 @@ cleanup
 
 tpm2_createek -Q -H 0x8101000b -g rsa -p ek.pub
 
-tpm2_createak -Q -E 0x8101000b  -k 0x8101000c -g rsa -D sha256 -s rsassa -p ak.pub  -n ak.name
+tpm2_createak -Q -E 0x8101000b -k 0x8101000c -g rsa -D sha256 -s rsassa -p ak.pub -n ak.name
+
+# Find a vacant persistent handle
+tpm2_createak -E 0x8101000b -k - -g rsa -D sha256 -s rsassa -p ak.pub -n ak.name > ak.log
+phandle=`grep "ak-persistent-handle: " ak.log | awk '{print $2}'`
```

Comment:
I like to get better validation that this is indeed YAML compliant. I've been writing HERE doc python scripts that use the yaml module to import the output. I think it's in helpers.sh. You may need to add a new routine if your format doesn't follow those.

The only time we don't check, is if the tool has simple yaml scalar output, for example:
```
tpm2_xxx
42
```
That's a scalar output in YAML.

Reply:
yaml_verify from helpers.sh works for the output of the tools I'm modifying here, thanks for pointing it out.